src/pages/installing/uninstalling.md

```diff
@@ -6,6 +6,8 @@ slug: uninstalling
 toc: true
 ---
 
+_Note: Refer here for uninstalling instructions for [IBM Annotator for Clinical Data Container Edition](/installing/uninstalling-ibm/)._ 
```

Comment:
Move this down, after the web console steps to uninstall. Then say something like "To uinstall an ACD instance that was installed with IBM's cloudctl CLI, refer here ..."